wandb/__init__.py

```diff
@@ -784,6 +784,7 @@ def init(job_type=None, dir=None, config=None, project=None, entity=None, reinit
         force (bool, optional): Force authentication with wandb, defaults to False
         magic (bool, dict, or str, optional): magic configuration as bool, dict, json string,
             yaml filename
+        anonymous (str, optional): Can be "allow", "must", or "never". Controls whether anonymous logging is allowed.
```

Comment:
defaults to never

Reply:
(most of the time :) )
probably good enough to say never, the exceptions are if you do a wandb.login or set an environment variable.